apps/angular-mongodb-e2e/fixtures/editTutorials.test.js

```diff
@@ -0,0 +1,104 @@
+import {URLS} from '../utils/route.util';
+import {loginElements} from '../page-objects/login.po';
+import {password} from '../utils/constant.util';
+import {getUrl} from '../page-objects/editTutorials.po';
+import {
+  tutorialsElements,
+  updateToturial,
+  publishedTutorials,
+  UnPublishedTutorials,
+  deleteTutorials
+} from '../page-objects/editTutorials.po';
+
+fixture('Welcome For edit tutorials')
+  .page(URLS.LOGIN)
+
+  .beforeEach(async t => {
+    //Arrange
+    const userName = 'peter.flowers';
```

Comment:
make this global in this fixture and use it in the test
see: https://devexpress.github.io/testcafe/documentation/guides/basic-guides/organize-tests.html#share-variables-between-fixture-hooks-and-test-code